glide-core/redis-rs/redis/Cargo.toml

```diff
@@ -153,6 +153,7 @@ bigdecimal = ["dep:bigdecimal"]
 num-bigint = []
 uuid = ["dep:uuid"]
 disable-client-setinfo = []
+valkey-gte-8 = []
```

Comment:
That's are wrong usage of Rust's feature. A feature is a compile time property, not a runtime property. If you want to disable a test from being executed unless the engine is 8.0 and above, you should check it during **runtime**

Please remove this

Reply:
Thanks @eifrah-aws for supporing me there

Reply:
@eifrah-aws there is no disagreement on it, but occasionally, there's a need to prioritize. It's unprofessional to release with a broken CI system, would you block on that as well? Everyone has his baby, but creating a whole test utils platform 2 days after freeze is not align with team goal.
 
This is not runtime, it's test compile.
There are bigger problems in the tests platform of redis-rs and glide core, can we block PRs on them as well?

@adarovadya opened an issue about it, and assigned herself. It affects CI only, which currently is always my problem anyway. And it can be removed after freeze calmly, no need to take her to the edge and make her work throw the weekend and night to make deadlines on non super-critical issues.

I think we need to have better management. Every so often we make a giant change, like moving an entire package into the repo overnight, and deciding to do it in iterations instead of clean and arrange.
But on nit things, we will block a PR when dev is with back to the wall.
Really, don't agree with that.

Reply:
to check it in runtime we need in every CI setup, to create a cluster and run the INFO command to determine the version. Is there an alternative or more efficient way to retrieve the version information?

Reply:
The way to do it will be to have in the tests platform a function that can retrieve the version of the cluster, and skip the test base on it. We didn't have such cases in the core/r-rs before, so what we have in the other clients doesn't exist.
The fast way to do it will be in your tests, create a client and use it to check, and skip if the version doesn't is not 8.0.
The right way is to add it to the textContext as a call to the env before creating server and using good semver to make it fits for the next version to come.
Without starting to actually creating server, and client in the tests, than skip base on constant.
It's not super complex, but it's a separate PR.
Its testing platform, not AZ.

Reply:
resolved. I created a static function that creates a cluster and saves its version. The cluster will only be created  for the first function call. After that, the version will be saved in a static variable.